.github/workflows/ci.yaml

```diff
@@ -0,0 +1,48 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# This workflow will build a Java project with Maven
+# See also:
+#   https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: CI
+
+on:
+  push:
+    branches: [ '*' ]
+  pull_request:
+    branches: [ '*' ]
+
+jobs:
+  build:
+    name: ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, macOS-latest, ubuntu-20.04]
+    timeout-minutes: 10
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-python@v2
+      with:
+        python-version: '3.6'
+    - name: Install required packages
+      run: pip install -r ./lib/requirements.txt
```

Comment:
Does the earlier setup-python action setup pip as pip3?

Reply:
Empirically, looks fine. I did a small test elsewhere and it seems to explicitly pick up python and pip correctly from `/opt/hostedtoolcache/Python/3.6.12/x64/bin`

Reply:
Hi @keith-turner I went through [docs](https://docs.github.com/en/free-pro-team@latest/actions/guides/building-and-testing-python) and think the usage of `pip` is fine. The `setup-python` action does adjust `PATH` accordingly to ensure the correct versions are picked up. If you see any other issue or have another concern, do let me know.